DeviceBridge/Services/BridgeService.cs

```diff
@@ -21,21 +21,21 @@ public BridgeService(IConnectionManager connectionManager)
         public async Task SendTelemetry(Logger logger, string deviceId, IDictionary<string, object> payload, CancellationToken cancellationToken, IDictionary<string, string> properties = null, string componentName = null, DateTime? creationTimeUtc = null)
         {
             logger.Info("Sending telemetry for device {deviceId}", deviceId);
-            await _connectionManager.AssertDeviceConnectionOpenAsync(deviceId, true /* temporary */, false, cancellationToken);
+            await _connectionManager.AssertDeviceConnectionOpenAsync(deviceId, true /* temporary */, cancellationToken);
```

Comment:
What is false parameter used for in previous version?

Reply:
This option controlled if it would try to reopen a failed connection or just give up if a previous connection was in a failed state.
Now we always retry failed connections, so the user doesn't need to manually retry (if you send a telemetry and the connection was failed, it will try to open again)